planning/behavior_path_planner/src/scene_module/lane_change/util.cpp

```diff
@@ -129,7 +129,7 @@ std::vector<LaneChangePath> getLaneChangePaths(
       minimum_lane_change_prepare_distance);
     const double lane_changing_distance = getDistanceWhenDecelerate(
       lane_change_prepare_velocity, acceleration, lane_changing_duration,
-      minimum_lane_change_length + buffer);
+      minimum_lane_change_length);
```

Comment:
At L159, it is still `minimum_lane_change_length + buffer`. Is that correct?
```
      s_end = std::min(s_end, lane_length - num * (minimum_lane_change_length + buffer));
```

Reply:
yes, that is correct.

Referring to the image below.
The L132 as you commented only compute the lane changing distance (only the green part)
While L159, computes purple and green part. 
![image](https://user-images.githubusercontent.com/93502286/189023104-71209351-8d03-4146-91dd-3d40b75df219.png)

Reply:
Got it, thank you!!